cmd/generate-hashes/generate-hashes.go

```diff
@@ -31,16 +39,36 @@ func main() {
 		os.Exit(1)
 	}
 
+	// set up hasher
+	hashFunc := func() hash.Hash {
+		return sha3.NewLegacyKeccak256()
+	}
+	hashPool := legacy.NewTreePool(hashFunc, 128, legacy.PoolSize)
```

Comment:
please comment that 128 is only true for unencrypted chunks

Reply:
Well, while that's true from a functional perspective, it's not quite true here. The chunk is still 4096 bytes and segment size is determined by the hasher. There's no other way to tell the BMT that it's 32*128, unless we mess with the underlying hasher.

In fact, I believe the bmt should be agnostic to all of this. It should care about hashing digest-size segments together and that's it.